Correct usage of ellipsis, part 6 (#18204)
mdn/content: files/en-us/web/javascript/reference/global_objects/intl/index.md

5. (optionally) one or more BCP 47 extension sequences, and
6. (optionally) a private-use extension sequence

> With all present subtags and sequences separated by hyphens. Locale identifiers are case-insensitive ASCII. However, it's conventional to use title case (first letter capitalized, successive letters lower case) for script subtags, upper case for region subtags, and lower case for everything else. For example:

Reviewer: ```suggestion
Each subtag and sequence are separated by hyphens. Locale identifiers are case-insensitive ASCII. However, it's conventional to use title case (the first letter is capitalized, successive letters are lower case) for script subtags, upper case for region subtags, and lower case for everything else. For example:
```
